docs/ingest/otel-sinks/fluent-bit.mdx

```diff
@@ -0,0 +1,4 @@
+# Integration With OTEL Providers
+
+## Integration With Fluent Bit
```

Comment:
Neat! There might be two things to highlight:

- Ingesting directly into Postgres
- Ingesting into S3/etc. object storage and querying to ParadeDB via `pg_analytics`, our ingest API.

I suspect especially in the case of log ingestion, many will want to send directly to S3 for lower cost. We should highlight both.

Reply:
You can flush to S3 via a cron job. We ship `pg_cron` in ParadeDB.